EightMinuteEmpireLegends/Map/MapDriver.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+#include <string>
+#include <vector>
+#include "Map.h"
+
+// Returns string list of a vertex vector
+std::string vertices_toString(std::vector<Vertex*> v) {
```

Comment:
Why do you have implementation details in a header file?

Reply:
Those are just functions I use exclusively in the driver class to help visualize data. I didn't want to include their definitions in the cpp file because I wanted to limit the clutter.

Reply:
ok makes sense

Reply:
I disagree.
Should be in its own utils file.